File: 2-4/main.js
const kisetsu = +process.argv[2]

switch (kisetsu){
    case 2:
    case 3:
    case 4:
      console.log('春');
      break;
    case 5:
    case 6:
    case 7:
      console.log('夏');
      break;
    case 8:
    case 9:
    case 10:
      console.log('秋');         
      break;
    default:
      console.log('冬');
  }

File: 2-4/main.py
#-*- coding: utf-8 -*-
 
import sys

x = int(sys.argv[1])

if 2 <= x <= 4:
  print("春")

elif 5 <= x <= 7:
  print("夏")

elif 8 <= x <= 10:
  print("秋")

elif 1 == x and 11 <= x:
  print("冬")